test/controllers/util/FormFieldSpec.scala

```diff
@@ -513,6 +562,28 @@ class FormFieldSpec extends AnyWordSpec with MockitoSugar with BeforeAndAfterEac
       })
     }
 
+    "updateError should set an error message as per the given error code" in {
```

Comment:
Can't see a check for the max character limit error in this test?

Reply:
Actually, the date field will never have a max character limit error. It is only applicable for `TextArea` and `RadioButtonGroupField` (dependency).

Reply:
Fair enough